templates/properties/unit_detail.html

```diff
@@ -1,28 +1,61 @@
-    <p>
-        <strong>Property:</strong> {{ unit.property.name }}
-    </p>
-    <p>
-        <strong>Location:</strong> {{ unit.property.sub_locality.name }}, {{ unit.property.city.name }}, {{ unit.property.state.name }}, {{ unit.property.country.name }}
-    </p>
-    <p>
-        <strong>Description:</strong> {{ unit.description }}
-    </p>
-    <p>
-        <strong>Number of Beds:</strong> {{ unit.num_beds }}
-    </p>
-    <p>
-        <strong>Number of Bathrooms:</strong> {{ unit.num_bathrooms }}
-    </p>
-    <p>
-        <strong>Number of Kitchens:</strong> {{ unit.num_kitchens }}
-    </p>
-    <p>
-        <strong>Number of Living Rooms:</strong> {{ unit.num_living_rooms }}
-    </p>
-    <p>
-        <strong>Number of Stores:</strong> {{ unit.num_stores }}
-    </p>
-{% endblock content %}
+    <link rel="stylesheet"
+          href="https://unpkg.com/photoswipe@5.2.2/dist/photoswipe.css" />
+    <div class="container mt-4">
+        <div class="row">
+            <div class="col-md-6">
+                <h3>Image Gallery</h3>
+                <div class="pswp-gallery" id="my-gallery">
+                    {% for image in unit.images.all %}
+                        <a href="{{ image.image.url }}"
+                           data-pswp-width="{{ image.image.width }}"
+                           data-pswp-height="{{ image.image.height }}"
+                           target="_blank">
+                            <img src="{{ image.image.url }}" alt="" />
+                        </a>
+                    {% endfor %}
+                </div>
+            </div>
+            <div class="col-md-6">
+                <h2>Unit Details: {{ unit.unit_number }}</h2>
+                <p>
+                    <strong>Property:</strong> {{ unit.property.name }}
+                </p>
+                <p>
+                    <strong>Location:</strong> {{ unit.property.sub_locality.name }}, {{ unit.property.city.name }}, {{ unit.property.state.name }}, {{ unit.property.country.name }}
+                </p>
+                <p>
+                    <strong>Description:</strong> {{ unit.description }}
+                </p>
+                <p>
+                    <strong>Number of Beds:</strong> {{ unit.num_beds }}
+                </p>
+                <p>
+                    <strong>Number of Bathrooms:</strong> {{ unit.num_bathrooms }}
+                </p>
+                <p>
+                    <strong>Number of Kitchens:</strong> {{ unit.num_kitchens }}
+                </p>
+                <p>
+                    <strong>Number of Living Rooms:</strong> {{ unit.num_living_rooms }}
+                </p>
+                <p>
+                    <strong>Number of Stores:</strong> {{ unit.num_stores }}
+                </p>
+            </div>
+        </div>
+    </div>
+    <script type="module">
+    import PhotoSwipeLightbox from 'https://unpkg.com/photoswipe/dist/photoswipe-lightbox.esm.js'
```

Comment:
his code will be require on several pages where we want to show gallery, make it modular so it could be reused instead of duplication.